terraform/cloudwatch.tf.json.template.py

```diff
@@ -0,0 +1,72 @@
+from azul.template import emit
+from azul import config
+
+if config.terraform_cloudwatch_alarms:
+    emit({
+        "resource": [
+            {
+                "aws_cloudwatch_metric_alarm": {
+                    "CPUUtilization": {
+                        "alarm_name": config.es_domain+"-CPUUtilization",
+                        "actions_enabled": True,
+                        "comparison_operator": "GreaterThanOrEqualToThreshold",
+                        "evaluation_periods": "2",
+                        "metric_name": "CPUUtilization",
+                        "namespace": "AWS/ES",
+                        "period": "3600",
+                        "statistic": "Average",
+                        "threshold": "85",
+                        "alarm_description": "This metric monitors ES CPU utilization",
+                        "dimensions": {
+                            "ClientId": "${data.aws_caller_identity.current.account_id}",
```

Comment:
Note that we also have `azul.deployment.aws.account`. I think we should use that instead. For example usage look at:

https://github.com/DataBiosphere/azul/blob/develop/terraform/backend.tf.json.template.py#L10